docs/Manual_Installation.md

```diff
@@ -0,0 +1,109 @@
+# Manual Installation
+
+These instructions are for people who want to run klipper on
+custom system that's not debian-based. Flashing steps are same
+as in [Installation](Installation.md) instruction.
+Naturally this instruction is for more advanced users.
+
+# Preparing for installation
+
+First you need to install required packages. This step is different
+for every other package manager and distribution so provided are
+debian packages and fiding package names for your distribution are
```

Comment:
'fiding' should be finding

Reply:
Will fix but i'll delay in case you find more. Thanks for help!